docs/setup.md

```diff
@@ -11,82 +11,95 @@
+- **ServiceNow** - ServiceNow connector provides functionality to create, read, update, and delete records of type Table and Catalog. To configure and use the ServiceNow connector, refer to [Configuring ServiceNow](https://docs.fortinet.com/fortisoar/connectors/servicenow-v2-0-1)
+- **Splunk** - Splunk connector allows users to invoke search, fetch events to related search, invoke alert actions, update notables, sync Splunk users to FortiSOAR, etc. To configure and use the Splunk connector, refer to [Configuring Splunk](https://docs.fortinet.com/fortisoar/connectors/splunk_new)
+- **ElasticSearch** - ElasticSearch is a distributed, RESTful search and analytics engine capable of solving several use cases. At the heart of the Elastic Stack, it centrally stores your data so you can discover the expected and uncover the unexpected. To configure and use the ElasticSearch connector, refer to [Configuring ElasticSearch](https://docs.fortinet.com/fortisoar/connectors/)
 
 ## Setup Outbreak Response Framework on FortiSOAR
-After installation of **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured "Threat Detection Integrations" and "Ticketing/ITSM Integrations" on FortiSOAR.
+After installation of the **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured "Threat Detection Integrations" and "Ticketing/ITSM Integrations" on FortiSOAR.
 
-1. Log in to FortiSOAR and [after installation](#installation), click the button **Configure** from the lower-left of the screen.
+1. Log in to FortiSOAR, after [installation](#installation) completes.
+2. Click the button **Configure** from the lower-left of the screen.
 
     ![Outbreak Response start configuration](./res/config-wizard-00.png)
 
-2. Click the button **Let's get started** on the Outbreak Response Framework configuration page.
+3. Click the button **Let's get started** on the Outbreak Response Framework configuration page.
 
     ![Outbreak Alert get started](./res/config-wizard-01.png)
 
-3. Specify Threat Detection Integrations sources to run outbreak response hunt activities. Click **Next** to proceed.
+4. Select Threat Detection Integration sources to run outbreak response hunt activities and click **Next**.
 
     ![Define your Threat Detection Integrations](./res/config-wizard-02.png)
 
-4. Specify Ticketing/ITSM Integrations sources as part of your response or threat management strategy. Click **Next** to proceed.
+5. Select Ticketing/ITSM Integration sources as part of your response or threat management strategy and click **Next**.
```

Comment:
same a above. Intent of having this option is important here.

Reply:
Done